Intersect.Server.Core/Core/MapInstancing/InstanceProcessor.cs

```diff
@@ -0,0 +1,74 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Intersect.Server.Core.MapInstancing.Controllers;
+using Intersect.Server.Entities;
+using Intersect.Server.Maps;
+
+namespace Intersect.Server.Core.MapInstancing;
+public static class InstanceProcessor
+{
+    private static Dictionary<Guid, InstanceController> InstanceControllers = new Dictionary<Guid, InstanceController>();
+
+    public static Guid[] CurrentControllers => InstanceControllers.Keys.ToArray();
+
+    public static bool TryGetInstanceController(Guid instanceId, out InstanceController controller)
+    {
+        return InstanceControllers.TryGetValue(instanceId, out controller);
+    }
+
+    private static void CleanupOrphanedControllers(List<MapInstance> activeMaps)
```

Comment:
Change this to `IEnumerable<MapInstance>`

Reply:
Curious, why? But done (again, slightly differently)